apps/omg/test/omg/state/core_test.exs

```diff
@@ -680,23 +680,29 @@ defmodule OMG.State.CoreTest do
   @tag fixtures: [:alice, :state_alice_deposit, :state_empty]
   test "given exit infos in various forms translates to utxo positions",
        %{alice: alice, state_alice_deposit: state, state_empty: state_empty} do
+    # TODO: Fix this !@#%
     # this test checks whether all ways of calling `get_exiting_utxo_positions/2` translates
     # to given exiting utxo positions
-    utxo_pos_exits = [Utxo.position(@blknum1, 0, 0), Utxo.position(@blknum1, 0, 1)]
+    expected_utxo_pos_exits = [{:utxo_position, @blknum1, 0, 0}, {:utxo_position, @blknum1, 0, 1}]
 
-    assert utxo_pos_exits ==
+    utxo_pos_exits = [
+      %{blknum: @blknum1, txindex: 0, oindex: 0},
+      %{blknum: @blknum1, txindex: 0, oindex: 1}
+    ]
+
+    assert expected_utxo_pos_exits ==
              utxo_pos_exits
-             |> Enum.map(&%{call_data: %{utxo_pos: Utxo.Position.encode(&1)}})
+             |> Enum.map(&%{call_data: %{utxo_pos: ExPlasma.Utxo.pos(&1)}})
```

Comment:
For this usage, It is seamless to use the `ExPlasma.Utxo` library directly since we want to generate values such as the integer input utxo position instead of having a wrapper around it.